eox_nelp/openedx_filters/xapi/filters.py

```diff
@@ -0,0 +1,53 @@
+"""This file contains all the NELP filters that can be implemented by the event-routing-backends library
+
+Filters:
+    XApiActorFilter: Modifies the standard ACtor in order to include the name attribute.
+"""
+from django.contrib.auth import get_user_model
+from openedx_filters import PipelineStep
+from tincan import Agent
+
+User = get_user_model()
+
+
+class XApiActorFilter(PipelineStep):
```

Comment:
This class could say something in the name  that we are trying to fit and actor for Nelp conditions...

Reply:
That is not explicit but adding something like that could be redundant, I mean this is eox-nelp the plugin which extends the platform y the NELP way